osbuild/util/ostree.py

```diff
@@ -160,15 +160,15 @@ def rev_parse(repo: PathLike, ref: str) -> str:
     return msg
 
 
-def show(repo: PathLike, checksum: str) -> str:
+def show(repo: PathLike, checksum: str, *args) -> str:
```

Comment:
this might be one where they will ask us in the code review to just call `ostree.cli()` to get the `--print-metadata-key=version` instead of changing this function signature.

Maybe we should even revert back to that anyway.

Reply:
Yeah, I just thought I should utilize the tools already provided, but if it requires us to add the `*args` parameter, it might just be better the call using `ostree.cli()` to make the PR less intrusive on the existing code. I think it was just shame that accepting additional flags wasn't part of the original function, but I'm cool with just calling the command through `ostree.cli()`